pkg/scheduler/framework/session_plugins.go

```diff
@@ -227,6 +228,26 @@ func (ssn *Session) Reclaimable(reclaimer *api.TaskInfo, reclaimees []*api.TaskI
 	return victims
 }
 
+// getJobStartTime returns the earliest start time of any task in the job
+func getJobStartTime(job *api.JobInfo) time.Time {
```

Comment:
/cc @jjuliogallegos01
Does this meet your requirements? @GautamBytes takes the earliest running Pod under the job as the starttime. Because the scheduler cannot get the vcjob information, it may have to rely on vc-controller to attach the job information to the podgroup so that the scheduler can perceive it.